rclcpp_components/test/test_component_manager_api.cpp

```diff
@@ -321,3 +327,9 @@ TEST_F(TestComponentManager, components_api)
     }
   }
 }
+
+TEST_F(TestComponentManager, components_api)
+{
+  test_components_api(false);
```

Comment:
It's nicer to use a parameterized test for this, see https://github.com/google/googletest/blob/main/docs/advanced.md#value-parameterized-tests.

If you feel that's an overkill here, add [scoped traces](https://github.com/google/googletest/blob/main/docs/advanced.md#adding-traces-to-assertions) so it's more clear where the errors come from.
You could also return an [AssertionResult](https://github.com/google/googletest/blob/main/docs/advanced.md#adding-traces-to-assertions) to avoid [this pitfall](https://github.com/google/googletest/blob/main/docs/advanced.md#propagating-fatal-failures).